server/api/CMakeLists.txt

```diff
@@ -150,6 +151,10 @@ add_library(
   "${CMAKE_CURRENT_SOURCE_DIR}/src/rsUserAdmin.cpp"
   "${CMAKE_CURRENT_SOURCE_DIR}/src/rsZoneReport.cpp"
 )
+add_dependencies(
+  irods_server_api
+  irods_genquery2_parser
+)
```

Comment:
This was necessary for allowing compiling from scratch to succeed.

Reply:
@SwooshyCueb Any opinions on this?

Reply:
This is odd, but fine for now. It'll get cleaned up as I move us toward better CMake.

Reply:
Sounds good. Marking as resolved.